### PR TITLE
1. Modified TextInterfaceVuew to correctly print board size above 10 …

### PR DESCRIPTION
**Pull request by scherjo**
1. Modified TextInterfaceView to correctly print board size above 10 (up to 26)
2. Modified TextInterfaceController to more appropriately parse input points (and work correctly for arbitrarily large board sizes)
3. Fixed bug in Player that caused boards to use default size rather than correct size

### CODE REVIEW

**Review by jgao222 — APPROVED** [review]
Looks good.

**Comment by jgao222** [comment]
Might want to change some of the other places we parse input as points to use the new `parsePoint` method, like in the ship placement.

File: app/src/main/java/textInterface/TextInterfaceController.java [hunk 1]
```diff
@@ -337,12 +339,70 @@ private int getShipLength() {
...
+     * private method to return the number of letters for a valid
+     * input for the game (based on the game's size)
+     */
+    private int numLetters() {
+        int boardSize = game.size();
+        boardSize--;
+        int numLetters = 0;
+        while (boardSize > 0) {
+            numLetters++;
+            boardSize /= 26;
```
Comment (edward-qin) [inline app/src/main/java/textInterface/TextInterfaceController.java:379]: Is this method just (game.size() - 1)/26?

**Review by edward-qin — APPROVED** [review]
looks good

**Pull request merged**